src/drivers/imx/ipc.c

```diff
@@ -125,16 +125,16 @@ static void ipc_platform_complete_cmd(void *data)
 	}
 }
 
-void ipc_platform_send_msg(struct ipc *ipc)
+void ipc_platform_send_msg(void)
```

Comment:
Could you enhance the commit message a bit and explain why you are removing the argument. It seems we have a static _ipc instance already defined, but why we are changing now, that would be good to explain in the commit message.

Reply:
Done.